src/Flex.php

```diff
@@ -161,11 +160,44 @@ public function update(Event $event): void
         }
 
         $this->io->writeError(sprintf('<info>Symfony operations: %d recipe%s</>', count($recipes), count($recipes) > 1 ? 's' : ''));
+        $installContribs = $this->composer->getPackage()->getExtra()['symfony']['allow-contrib'] ?? false;
         foreach ($recipes as $recipe) {
-            if ($recipe->isNotInstallable() && $recipe->isEmpty()) {
-                $this->io->writeError(sprintf('  - <warning>Ignoring</> recipe %s', $this->formatOrigin($recipe->getOrigin())));
-                $this->io->writeError('    Enable via composer config extra.symfony.allow-contrib true');
-                continue;
+            if ('install' === $recipe->getJob() && !$installContribs && $recipe->isContrib()) {
+                $this->io->writeError(sprintf('  - <warning>WARNING</> %s', $this->formatOrigin($recipe->getOrigin())));
+                $question = '    This package wants to update your project for better integration.
```

Comment:
This doesn't make it look like it's clear that the issue is that this comes from the contrib repository. It's tough, we want to explain what's going on ("there are 2 repos, one is super protected, the other is not as protected, and this comes from that repo")... without all those words :).

Reply:
Indeed, the current sentence is far from perfect... I was expecting some help from you :)

Reply:
Indeed :). I don't remember if I was short on time... or just being lazy... Let's just pretend it was the first

>    The recipe for this package comes from the contrib repository, which is open to community contributions.
>    Allow this recipe?

Later, we could even add a short link to some docs (e.g. `http://symfony.com/contrib-recipes`)

Reply:
I've used:

```
The recipe for this package comes from the "contrib" repository, which is open to community contributions.
    Do you want to execute this recipe?
```